packages/core/src/read/impl.js

```diff
@@ -119,13 +120,23 @@ export async function performRead(context, readParams) {
 
       const enrichedResponse = await enrichment(readValue, enrichContext)
 
-      const transformContext = {
+      const enhanceContext = {
```

Comment:
As far as I understand, here, enhancers are run **after** the enrichers.
To me, it makes more sense for enhancers to be run **before** enrichers, because:
- They are a more restricted way of transformation, which only works on root elements
- Root (parent elements) might decide to alter children; it makes sense for parent transformations to be run before the children's (registered via enrichers, for example)